docs/networking/calico.md

```diff
@@ -54,13 +54,18 @@ To enable this mode in a cluster, add the following to the cluster spec:
   networking:
     calico:
       crossSubnet: true
+      awsSrcDstCheck: Disable
 ```
 
+> This is a deprecated behavior, to disable source/destination address checks use networking.calico.awsSrcDstCheck  
+> If crossSubnet was set to true, ([k8s-ec2-srcdst](https://github.com/ottoyiu/k8s-ec2-srcdst)) resources exists on master nodes which can be removed manually  
+
 In the case of AWS, EC2 instances have source/destination checks enabled by default.
 When you enable cross-subnet mode in kops, an addon controller ([k8s-ec2-srcdst](https://github.com/ottoyiu/k8s-ec2-srcdst))
 will be deployed as a Pod (which will be scheduled on one of the masters) to facilitate the disabling of said source/destination address checks.
 Only the masters have the IAM policy (`ec2:*`) to allow k8s-ec2-srcdst to execute `ec2:ModifyInstanceAttribute`.
 
+
```

Comment:
````suggestion
```

In the case of AWS, EC2 instances have source/destination checks enabled by default.
When you enable cross-subnet mode in kops 1.19+, it is equivalent to: 
```yaml
  networking:
    calico:
      awsSrcDstCheck: Disable
      IPIPMode: CrossSubnet
```
An IAM policy (`ec2:*`) will be added to all nodes to allow Calico to execute `ec2:DescribeInstances` and `ec2:ModifyNetworkInterfaceAttribute`, as required when [awsSrcDstCheck](https://docs.projectcalico.org/reference/resources/felixconfig#spec) is set.

For older versions of kops, an addon controller ([k8s-ec2-srcdst](https://github.com/ottoyiu/k8s-ec2-srcdst))
will be deployed as a Pod (which will be scheduled on one of the masters) to facilitate the disabling of said source/destination address checks.
Only the control plane nodes have the IAM policy (`ec2:*`) to allow k8s-ec2-srcdst to execute `ec2:ModifyInstanceAttribute`.
````

Reply:
@hakman the policy getting added for awsSrcDstCheck is not `ec2:*` its `ec2:DescribeInstances` and `ec2:ModifyNetworkInterfaceAttribute`

Reply:
True, I copy/pasted from below. Updated.